### PR TITLE
[script.module.dateutil@matrix] 2.8.2+matrix.1

### PR DESCRIPTION
**Pull request by razzeee**
### Add-on details:

- **General**
  - Add-on name: dateutil
  - Add-on ID: script.module.dateutil
  - Version number: 2.8.2+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/Razzeee/script.module.dateutil
  
Packed for Kodi from https://github.com/dateutil/dateutil

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: script.module.dateutil/addon.xml [hunk 1]
```diff
@@ -1,22 +1,20 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (razzeee) [inline script.module.dateutil/addon.xml:1]: Seems like someone changed this without sending the changes upstream

File: script.module.dateutil/addon.xml [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
 <addon id="script.module.dateutil"
-       name="python-dateutil"
-       version="2.8.1+matrix.1"
-       provider-name="Paul Ganssle, Gustavo Niemeyer">
+       name="dateutil"
+       version="2.8.2+matrix.1"
+       provider-name="Gustavo Niemeyer, Tomi Pieviläinen, Yaron de Leeuw, Razzeee">
```
Comment (enen92) [inline script.module.dateutil/addon.xml:5]: Have you patched the module or is it plain upstream? If it's plain upstream no need to define as an author

File: script.module.dateutil/addon.xml [hunk 3]
```diff
@@ -1,22 +1,20 @@
...
-    <license>Apache-2.0, BSD-3-Clause</license>
-    <website>https://dateutil.readthedocs.io/en/stable/</website>
-    <source>https://github.com/dateutil/dateutil</source>
-    <assets>
-      <icon>icon.png</icon>
-    </assets>
+    <summary lang="en_GB">dateutil - powerful extensions to datetime</summary>
+    <description lang="en_GB">Packed for Kodi from https://github.com/dateutil/dateutil</description>
+    <platform>all</platform>
+    <license>Revised BSD License</license>
```
Comment (enen92) [inline script.module.dateutil/addon.xml:16]: Spdx identifier if possible

File: script.module.dateutil/addon.xml [hunk 4]
```diff
@@ -1,22 +1,20 @@
...
+    <website>https://dateutil.readthedocs.org/en/latest/</website>
+    <source>https://github.com/Razzeee/script.module.dateutil</source>
```
Comment (enen92) [inline script.module.dateutil/addon.xml:18]: Source should point to the module repo if the module is not patched

File: script.module.dateutil/HOW_TO_RELEASE.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+1. Push to github wait for CI
```
Comment (enen92) [inline script.module.dateutil/HOW_TO_RELEASE.md:1]: Please remove from submission

Comment (razzeee) [reply script.module.dateutil/HOW_TO_RELEASE.md:1]: Already did, just haven't pushed yet

**Review by enen92 — APPROVED** [review]

**Pull request merged**